### PR TITLE
Add postgres init post command flag

### PR DESCRIPTION
**Pull request by msenmurugan**

### CODE REVIEW

File: pkg/blackduck/ctl_blackduck.go [hunk 1]
```diff
@@ -185,7 +186,8 @@ func (ctl *HelmValuesFromCobraFlags) AddCobraFlagsToCommand(cmd *cobra.Command,
 	cmd.Flags().StringVar(&ctl.flagTree.ExternalPostgresUserPassword, "external-postgres-user-password", defaults.ExternalPostgresUserPassword, "'user' password of external Postgres database")
 	cmd.Flags().StringVar(&ctl.flagTree.PostgresClaimSize, "postgres-claim-size", defaults.PostgresClaimSize, "Size of the blackduck-postgres PVC")
 	cmd.Flags().StringVar(&ctl.flagTree.AdminPassword, "admin-password", defaults.AdminPassword, "'admin' password of Postgres database")
-	cmd.Flags().StringVar(&ctl.flagTree.UserPassword, "user-password", defaults.UserPassword, "'user' password of Postgres database\n")
+	cmd.Flags().StringVar(&ctl.flagTree.UserPassword, "user-password", defaults.UserPassword, "'user' password of Postgres database")
+	cmd.Flags().StringVar(&ctl.flagTree.PostgresInitPostCommand, "postgres-init-post-command", defaults.PostgresInitPostCommand, "Postgres initialization post command. Supported. This flag is supported from Black Duck version 2020.8.0 and above\n")
```
Comment (mphammer) [inline pkg/blackduck/ctl_blackduck.go]: I think we should remove the "Supported." from the middle of the description

File: pkg/blackduck/ctl_blackduck_test.go [hunk 2]
```diff
@@ -257,6 +257,20 @@ func TestSetCRSpecFieldByFlag(t *testing.T) {
 			},
 		},
 		// case
+		{
+			flagName: "postgres-init-post-command",
+			changedCtl: &HelmValuesFromCobraFlags{
+				flagTree: FlagTree{
+					PostCommand: "",
```
Comment (mphammer) [inline pkg/blackduck/ctl_blackduck_test.go]: Can we put can actual value into here?

**Comment by mphammer** [comment]
We should probably add a check for version 2020.8.0 to this function - https://github.com/blackducksoftware/synopsysctl/blob/develop/pkg/blackduck/ctl_blackduck.go#L286

**Review by mphammer — CHANGES_REQUESTED** [review]
Just a few changes

File: pkg/blackduck/ctl_blackduck.go (hunk 1 above)
Comment (msenmurugan) [reply pkg/blackduck/ctl_blackduck.go]: done

File: pkg/blackduck/ctl_blackduck_test.go (hunk 2 above)
Comment (msenmurugan) [reply pkg/blackduck/ctl_blackduck_test.go]: done --> default is empty. customer will set only in case of Istio..

**Review by mphammer — APPROVED** [review]

**Pull request merged**